recipes-debian/util-linux/util-linux/run-ptest

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+
+# When udevd (from eudev) is running most eject/mount tests will fail because
+# of automount. We need to stop udevd before executing util-linux's tests.
+# The systemd-udevd daemon doesn't change the outcome of util-linux's tests.
+if [ -f /etc/init.d/udevd ]; then
```

Comment:
The eudev package use /etc/init.d/udev so It should be /etc/init.d/udev.

Reply:
fixed